tests/plugins/markers.py

```
@@ -0,0 +1,115 @@
+    # Skip destructive tests if in local mode and flag not set
+    if is_local_mode and not allow_destructive:
+        for item in items:
+            # Check if this test has ANY destructive marker
+            has_destructive_marker = any(
+                marker.name in destructive_markers
+                for marker in item.iter_markers()
+            )
+            
+            if has_destructive_marker:
+                reason = (
+                    "破坏性测试不允许在本机跑，使用 --allow-destructive 或 --ssh-host / --vm-deploy"
```

Ruff (0.15.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)